.circleci/config.yml

```diff
@@ -13,33 +13,6 @@ references:
     key: python-env-v1-{{ arch }}-{{ .Environment.CIRCLE_JOB }}-{{ checksum "setup.py" }}-{{ checksum "requirements/requirements-test.txt" }}-{{ checksum "requirements/requirements-precommit.txt" }}-{{ checksum ".circleci/ci-oldest-reqs.txt" }}-{{ checksum ".pre-commit-config.yaml" }}
 
 jobs:
-  pre-checks:
```

Comment:
I typically prefer to have the pre-commit checks to still run as part of the CI, to only run the more expensive tests in case that the source code is otherwise OK. Do you have a specific motivation to remove it?

Reply:
@csadorf See issue #623. We don't block on pre-checks because it increases our total CI time. pre-commit CI will push and automatically fix many of our style requirements, and we require pre-commit CI to pass before merging. https://github.com/glotzerlab/signac-flow/blob/da4e9c47903c59bfbc3a9b146631253931990a0c/.circleci/config.yml#L251-L254

Reply:
Ok.